lib/score_tracker.rb

```diff
@@ -27,6 +28,7 @@ def self.extract_rows(board)
 
   def self.are_any_arrays_full(arrays, character)
     is_an_array_full = false
+
     arrays.each do |array|
       if is_array_full(array, character)
         is_an_array_full = true
```

Comment:
You could just return true on line 34 and return false line 38 so you don't have to assign the value to a variable